binding-properties/src/main/java/com/iluwatar/bindingproperties/App.java

```diff
@@ -0,0 +1,51 @@
+package com.iluwatar.bindingproperties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is a simple test case for binding properties pattern.
+ */
+
+public class App {
+  private static final Logger LOGGER = LoggerFactory.getLogger(App.class);

```

Comment:
Use Lombok annotation

Reply:
It seems that the default logger generated by Lombok annotation @SLF4J is not named "log", I have no idea what it exactly named.

Reply:
You can check the other pattern implementations how it has been achieved there. To me, it seems to be named `LOGGER`.

Reply:
Well it seems that the default name is exactly LOGGER